project/Mima.scala

```diff
@@ -0,0 +1,10 @@
+import com.typesafe.tools.mima.core._
+import com.typesafe.tools.mima.core.ProblemFilters._
+
+object Mima {
+  val ignoredABIProblems: Seq[ProblemFilter] = {
+    Seq(
+      exclude[MissingClassProblem]("scalafix.package$XtensionRewriteCtx")
```

Comment:
After the v0.5.0 release, we can run `mimaReportBinaryIssues` in CI so that PRs will need to document the breaking changes in the public API.